reproducer.yml

```diff
@@ -57,11 +56,18 @@
       ansible.builtin.import_role:
         name: repo_setup
         tasks_from: rhos_release.yml
-
   roles:
     - role: ci_setup
-    - role: reproducer
 
+- name: Prepare switches if present
+  ansible.builtin.import_playbook: playbooks/switches_config.yml
+  when: "'switches' in groups"
```

Comment:
that will lead to "skipped" tasks. You want a pre_tasks in the `switch_config.yml` play imho, like we did for ceph.yml for instance.

Reply:
I've switched to this suggested approach.